### PR TITLE
Fix e2e test

### PR DESCRIPTION
**Pull request by Vincent056**
This PR is to fix TestKubeletConfigRemediation e2e test

### CODE REVIEW

**Comment by rhmdnd** [comment]
> This PR is to fix TestKubeletConfigRemediation e2e test

What was the issue?

**Comment by Vincent056** [comment]
> > This PR is to fix TestKubeletConfigRemediation e2e test
> 
> What was the issue?

There was some issue with the rule in the test content image

File: tests/e2e/e2e_test.go [hunk 1]
```diff
@@ -3557,7 +3529,7 @@ func TestE2E(t *testing.T) {
 
 				// We need to check that the remediation is auto-applied and save
```
Comment (rhmdnd) [inline tests/e2e/e2e_test.go:3530]: We could check that the check we're remediating failed before we attempt to fix it. I'm thinking about if the defaults change or for some reason the check starts to pass automatically, we wouldn't be testing the remediation, would we?

File: tests/e2e/e2e_test.go [hunk 2]
```diff
@@ -3492,24 +3460,28 @@ func TestE2E(t *testing.T) {
 						Description: "A test tailored profile to test kubelet remediation",
 						EnableRules: []compv1alpha1.RuleReferenceSpec{
 							{
-								Name:      prefixName(pbName, requiredRule),
+								Name:      "ocp4-kubelet-enable-streaming-connections",
+								Rationale: "To be tested",
+							},
+							{
+								Name:      "ocp4-version-detect-in-ocp",
```
Comment (rhmdnd) [inline tests/e2e/e2e_test.go:3467]: We're not checking the status of this below anywhere. Do we need it, or should we add an assertion for it?

File: tests/e2e/e2e_test.go (hunk 1 above)
Comment (Vincent056) [reply tests/e2e/e2e_test.go:3530]: If the check pass, it will not generate a remediation here

Comment (rhmdnd) [reply tests/e2e/e2e_test.go:3530]: Don't we want to test that it does?

Comment (Vincent056) [reply tests/e2e/e2e_test.go:3530]: I think it is being checked, see line #3523[https://github.com/ComplianceAsCode/compliance-operator/pull/197/files#diff-7d1fc1e232b55677582b51e2623aae6d8c15b5f0960ac6100fc7d733cb795dacR3523]

Comment (Vincent056) [reply tests/e2e/e2e_test.go:3530]: do you mean check the result of the first scan:
```
err = waitForSuiteScansStatus(t, f, namespace, suiteName, compv1alpha1.PhaseDone, compv1alpha1.ResultNonCompliant)
```

File: tests/e2e/e2e_test.go (hunk 2 above)
Comment (Vincent056) [reply tests/e2e/e2e_test.go:3467]: do you mean the checkResult, I think it is being check at the end, on line 3565,
```
				err = assertHasCheck(f, suiteName, scanName, checkResult)
```

File: tests/e2e/e2e_test.go (hunk 1 above)
Comment (rhmdnd) [reply tests/e2e/e2e_test.go:3530]: Aha - got it, thanks!

**Review by rhmdnd — APPROVED** [review]
/lgtm

File: tests/e2e/e2e_test.go (hunk 1 above)
Comment (Vincent056) [reply tests/e2e/e2e_test.go:3530]: the GitHub diff sometimes doesn't show the full code haha

Comment (Vincent056) [reply tests/e2e/e2e_test.go:3530]: thanks for the review!

**Comment by Vincent056** [comment]
add qe, px, and docs approve because it is a e2e test change

**Pull request merged**